Pod/Classes/BWWalkthroughViewController.swift

```diff
@@ -206,13 +212,17 @@ At the moment it's only used to perform custom animations on didScroll.
         
         // Constraints
         
-        let metricDict = ["w":vc.view.bounds.size.width,"h":vc.view.bounds.size.height]
+        // Create an explicit width constraint for the page view that is updated when the screen size changes
+        let widthConstraint = NSLayoutConstraint(item: vc.view, attribute: .Width, relatedBy: .Equal, toItem: nil, attribute: .NotAnAttribute, multiplier: 1, constant: vc.view.bounds.width)
+        // FIXME: Why doesn't an additional "-0-|" in the vertical visual format below work instead of the explicit height constraint?
+        let heightConstraint = NSLayoutConstraint(item: vc.view, attribute: .Height, relatedBy: .Equal, toItem: nil, attribute: .NotAnAttribute, multiplier: 1, constant: vc.view.bounds.height)
+        self.widthConstraints.append(widthConstraint)
+        self.heightConstraints.append(heightConstraint)
         
         // - Generic cnst
         
-        vc.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:[view(h)]", options:[], metrics: metricDict, views: ["view":vc.view]))
-        vc.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:[view(w)]", options:[], metrics: metricDict, views: ["view":vc.view]))
-        scrollview.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|-0-[view]|", options:[], metrics: nil, views: ["view":vc.view,]))
+        vc.view.addConstraints([ widthConstraint, heightConstraint ])
+        scrollview.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:|-0-[view]", options:[], metrics: nil, views: ["view":vc.view,]))
```

Comment:
Mistake here @ariok @knly. Instead of `"V:|-0-[view]"` has to be `"V:|-0-[view]|"`